borg/archiver.py

```diff
@@ -1514,6 +1553,102 @@ def build_parser(self, args=None, prog=None):
                                type=location_validator(archive=False),
                                help='path to the repository to be upgraded')
 
+        rewrite_epilog = textwrap.dedent("""
+        Rewrites the contents of existing archives.
+
+        --exclude, --exclude-from and PATH have the exact same semantics
+        as in borg create, this means if a PATH is specified the
+        rewrite includes that path and nothing else: PATH does *not* restrict
+        the rewrite to a path.
```

Comment:
this phrase is ambiguous.

i guess you did not mean:
this means if a PATH is specified the rewrite includes ( that path and nothing else ).

but rather:
this means if a PATH is specified the rewrite includes that path and it nothing mean anything else.